README.md

```diff
@@ -17,28 +17,29 @@ package the output of the transcoding job referenced by the message.
-| `AWS_SECRET_ACCESS_KEY`       | Optional AWS secret access key when `PACKAGE_OUTPUT_FOLDER` is an AWS S3 bucket                                                                                                                                                      |                          |
-| `S3_ENDPOINT_URL`             | Optional S3 Endpoint URL when `PACKAGE_OUTPUT_FOLDER` is an S3 bucket not on AWS                                                                                                                                                     |
+| `DASH_MANIFEST_NAME_TEMPLATE` | Template for name of DASH manifest file. Keywords `$INPUTNAME$` and `$JOBID$` will be replaced with basename of input, and id of encore job respectively. If unset, uses default from `shaka-packager-s3` library `manifest.mpd`      |                          |
+| `HLS_MANIFEST_NAME_TEMPLATE`  | Template for name of HLS manifest file. Keywords `$INPUTNAME$` and `$JOBID$` will be replaced with basename of input, and id of encore job respectively. If unset, uses default from `shaka-packager-s3` library, `index.m3u8`        |                          |
+| `ENCORE_PASSWORD`             | Optional password for the encore instance `user` user                                                                                                                                                                                 |                          |
+| `OSC_ACCESS_TOKEN`            | Optional OSC access token for accessing Encore instance in OSC                                                                                                                                                                        |                          |
+| `AWS_ACCESS_KEY_ID`           | Optional AWS access key id when `PACKAGE_OUTPUT_FOLDER` is an AWS S3 bucket                                                                                                                                                           |                          |
+| `AWS_SECRET_ACCESS_KEY`       | Optional AWS secret access key when `PACKAGE_OUTPUT_FOLDER` is an AWS S3 bucket                                                                                                                                                       |                          |
+| `S3_ENDPOINT_URL`             | Optional S3 Endpoint URL when `PACKAGE_OUTPUT_FOLDER` is an S3 bucket not on AWS                                                                                                                                                      |
+| `CALLBACK_URL`                | Optional callback service url. If enabled, the packager will send callbacks on packaging success or failure                                                                                                                           |
 
```

Comment:
Could we get `CALLBACK_USER` and `CALLBACK_PASSWORD` in here also?

Reply:
Fixed 👍